vars/packagepipelineUtils.groovy

```diff
@@ -242,4 +242,51 @@ class packagepipelineUtils implements Serializable {
         packagePipelineUtils.setScratchVars(parsedMsg)
     }
 
+    /**
+    * Based on tagMap, add comment to GH with
+    * instructions to manual commands
+    *
+    * @param map of tags
+    * @return
+    */
+    def sendPRCommentforTags(imageOperationsList) {
```

Comment:
So if you use the function from contra-lib Utils.groovy (vs the ones that have their own vars file), it would be like contraUtils.functionname .. you should see other examples of it in this file now since my other PR is merged